pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -470,6 +471,24 @@ func selectTrafficPolicyComponents(policy *networking.TrafficPolicy) (
 	loadBalancer := policy.LoadBalancer
 	tls := policy.Tls
 
+	var certFiles = []string{
+		"/etc/ssl/certs/ca-certificates.crt",                // Debian/Ubuntu/Gentoo etc.
+		"/etc/pki/tls/certs/ca-bundle.crt",                  // Fedora/RHEL 6
+		"/etc/ssl/ca-bundle.pem",                            // OpenSUSE
+		"/etc/pki/tls/cacert.pem",                           // OpenELEC
+		"/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem", // CentOS/RHEL 7
+		"/etc/ssl/cert.pem",                                 // Alpine Linux
+		"/usr/local/etc/ssl/cert.pem",                       // FreeBSD
+
+	}
+	if features.VerifyCertAtClient && tls.CaCertificates == "" {
+		for _, cert := range certFiles {
+			if _, err := os.Stat(cert); err == nil {
```

Comment:
This needs to be done in the proxy, not the control plane. The control plane can run on a completely distinct environment